package.json

```diff
@@ -37,7 +37,7 @@
   "scripts": {
     "lint": "semistandard",
     "test": "npm run lint && mocha --recursive test --reporter dot",
-    "start": "bin/stubby"
+    "start": "node bin/stubby.js"
```

Comment:
renaming this is a backward-incompatible change, as this changes the name of the executable on the PATH from `stubby` to `stubby.js`